src/Components/Light/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+import "./style.css"
+import bulbOn from "./images/light-on.svg";
+import bulbOff from "./images/light-off.svg";
+
+
+const Light = ({ name, state }) => {
+    const [stateState, setState] = useState(state)
```

Comment:
když máš 2 hodnoty, tak se typicky spáíš použije boolean. Takhle to tady jde a funguje, ale když použiješ boolean tak některé věci jdou zapsat lépe :)
```
const [stateState, setState] = useState(state === "on") // převedeme to co nám přijde na boolean
...
<div className="light" onClick={() => setState(!stateState) }} >
...
<img src={(stateState ? bulbOn : bulbOff} ></img>
```

Reply:
taky, většinou se stav pojmenovává tím co vyjadřuje, tj v tomhle případě bych to pojmenoval nějak jako
```
    const [isBulbOn, setBuldOn]
```